lib/metrics/services.js

```diff
@@ -282,6 +282,7 @@ module.exports = {
 	'sharecount': /https?:\/\/sharecount\.webservices\.ft\.com\//,
 	'smartology': /^https?:\/\/c\.smartology\.co\/matches\/pcid\/ft\.com\%252Fcontent\%252F/,
 	'spark-lists': /^https?:\/\/spark-lists\.ft\.com/,
+	'spark-lists-api': /^https:\/\/api\.ft\.com\/spark-lists/,
```

Comment:
It just came to my mind that since Spark lists is versioned (v1, v2), we could add the versioning to the id.What do you think?

```suggestion
	'spark-lists-api-v1': /^https:\/\/api\.ft\.com\/spark-lists\/v1/,
```

Reply:
@gyss Yeah, definitely looks like a good idea!